### ISSUE TITLE
Add build status, configure pipelines

### ISSUE THREAD

**Issue by fryette** [Softeq/ImagePicker-xamarin-ios#21]
Need to recreate build/release pipelines for azure.devops

**Comment by whitewolf33** [comment]
As part of #19 fix, v1.1.2 is released. The manual Archive build and upload to AppStore is successful and works fine. But when Azure DevOps is used, the build fails with the below error.

`MTOUCH : error MT2101: Can't resolve the reference 'AVFoundation.AVVideoCodecType[] AVFoundation.AVCapturePhotoSettings::get_AvailableEmbeddedThumbnailPhotoCodecTypes()', referenced from the method 'System.Void Softeq.ImagePicker.Media.Capture.PhotoCaptureSession/<>c__DisplayClass11_0::<CapturePhoto>b__0()' in 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'. [/Users/vsts/agent/2.153.2/work/1/s/iOS/Myproj.iOS.csproj]`

Once you setup the Azure Pipeline, I assume for the Sample project you will get the above error. Did not want to create a new issue as you have this open and would have found it anyway. Happy to test it and give you feedback once you have looked into this issue.

Thanks.

**Comment by wcoder** [comment]
Hi @whitewolf33, thanks for the report.

After some investigation, we have results:

- Currently, Azure DevOps by default used Xamarin iOS SDK version 11.14.0.13
- The latest available version is 12.8.0.2

For fix this issue need to add an additional task - bask script, for manually select a Xamarin SDK version:

```
sudo $AGENT_HOMEDIRECTORY/scripts/select-xamarin-sdk.sh 5_18_1
```

Before the build task.

More details:
- https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#how-can-i-manually-select-versions-of-tools-on-the-hosted-macos-agent
- https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-10.14-Readme.md#xamarin
